ocfweb/component/banner.py

```diff
@@ -0,0 +1,23 @@
+from collections import namedtuple
+from urllib.request import urlopen
+
+import yaml
+
+
+class BannerMessage(namedtuple('BannerMessage', [
+    'visible',
+    'force_lab_closed',
+    'html',
+])):
+    pass
+
+
+def get_banner_message():
+    """Get the front page banner message from the default location."""
+    with urlopen('file:///home/s/st/staff/banner.yaml') as f:
+        tree = yaml.load(f)
+    return BannerMessage(
+        visible=tree['visible'],
+        force_lab_closed=tree['force_lab_closed'],
+        html=tree['html'],
```

Comment:
I think eventually we might like to put some kind of wrapper (alert message styling) around the html so checking if it's visible might be useful.

```html
{% if banner.visible %}
   <div class="alert">{{banner.html | safe }}</div>
{% endif %}
```

As-is is fine, otherwise maybe use `None` instead of empty string?